library/std/Cargo.toml

```diff
@@ -92,6 +92,7 @@ backtrace = [
     'object/rustc-dep-of-std',
     'miniz_oxide/rustc-dep-of-std',
 ]
+backtrace-trace-only = []
```

Comment:
This will still compile addr2line, gimli and object. Maybe instead have a feature to enable just backtraces and then another feature which enables this feature and in addition enables backtrace symbolication?

Reply:
Yeah, doing this fully will require making changes to backtrace-rs, which I'm trying to avoid for the initial implementation. Also, although it's unstable, I don't want to break people currently using `build-std-features` just yet.